styleguide/SideNav.vue

```diff
@@ -1,15 +1,22 @@
 <template>
 	<div>
-		<ul>
-			<li
+		<img
+			src="/static/maker-logo.png"
```

Comment:
Needs to be a Webpack resolvable path (eg. relative or aliased) according to docs: https://vue-loader.vuejs.org/guide/asset-url.html#transform-rules